### PR TITLE
feat: content-type octet-stream support

### PR DESCRIPTION
**Pull request by ZorinArsenij**

### CODE REVIEW

File: internal/gen/_template/schemas.tmpl [hunk 1]
```diff
@@ -34,6 +34,13 @@ type {{ $.Name }} struct {
 {{- end }}
 {{/* Define struct */}}
 
+{{/* Define embedded */}}
+{{- else if $.Embedded }}
```
Comment (shadowspore) [inline internal/gen/_template/schemas.tmpl:38]: In this template we render types depending on ```$.Kind``` value.
```$.Embedded``` it's not Kind. This introduces inconsistency.

**Review by shadowspore — CHANGES_REQUESTED** [review]

File: internal/gen/_template/schemas.tmpl (hunk 1 above)
Comment (ernado) [reply internal/gen/_template/schemas.tmpl]: Merged `Embedded` and `RawBinary` kind into `Embedded` kind for consistency.

**Review by ernado — APPROVED** [review]

File: internal/oas/schema.go [hunk 2]
```diff
@@ -11,6 +11,9 @@ const (
 	Number  SchemaType = "number"
 	String  SchemaType = "string"
 	Boolean SchemaType = "boolean"
+
+	// Stream is custom type for raw byte stream.
+	Stream SchemaType = "x-stream"
```
Comment (shadowspore) [inline internal/oas/schema.go:16]: SchemaType contains types of an OpenAPI v3.0.3 JSON Schema.
```Stream``` it's not a part of that schema.
I think it should be detected and handled in ```gen``` pkg, not in the openapi parser.

**Review by shadowspore — CHANGES_REQUESTED** [review]

File: internal/oas/schema.go (hunk 2 above)
Comment (ernado) [reply internal/oas/schema.go:16]: Agree, will extract logic to gen.

File: internal/gen/_template/schemas.tmpl (hunk 1 above)
Comment (ZorinArsenij) [reply internal/gen/_template/schemas.tmpl]: Create `Stream` type for octet-stream

File: internal/oas/schema.go (hunk 2 above)
Comment (ZorinArsenij) [reply internal/oas/schema.go]: Remove `stream` from `oas` package

**Review by shadowspore — APPROVED** [review]

**Pull request merged**